ug/index.md

```diff
@@ -0,0 +1,23 @@
+<frontmatter>
+  header: header.md
+  pageNav: 2
+  pageNavTitle: "Chapters of This Page"
+  siteNav: ug-nav.md
+  footer: footer.md
+</frontmatter>
+
+# Welcome to the CATcher user guide
+
+CATcher is a desktop application for Crowd-sourced Anonymous Testing software. It uses GitHub as the backend for hosting bug reports.
```

Comment:
Minor grammar fix:

```suggestion
CATcher is a desktop application for Crowd-sourced Anonymous Testing of software. It uses GitHub as the backend for hosting bug reports.
```

Reply:
Alright!

Reply:
Sorry, can you also change `... is a desktop application` to `...is a cross platform application` ?

Reply:
Oops :x missed that out .. alright